tools/package-source.js

```diff
@@ -1712,6 +1711,37 @@ _.extend(PackageSource.prototype, {
     return _.keys(packages);
   },
 
+  // Returns an array of objects, representing this package's public
+  // exports. Each object has the following keys:
+  //  - name: export name (ex: "Accounts")
+  //  - arch: an array of strings representing architectures for which this
+  //    export is declared.
+  //
+  // This ignores testOnly exports.
+  getExports: function () {
+    var self = this;
+    var ret = {};
+    // Go over all of the architectures, and aggregate the exports together.
+    _.each(self.architectures, function (arch) {
+      _.each(arch.declaredExports, function (exp) {
+        // Skip testOnly exports -- the flag is intended for use in testing
```

Comment:
I mean it definitely was an intentional feature, but we have not found a use for it other than in test-helpers.  (Dunno about non-core packages.)

In fact, looking at the use of testOnly in our code, I suggest we should deprecate the TestOnly feature and just give every package a special symbol with a name like TestExports or something that is only exported to its own test package.  (Every other use of testOnly is to export a single symbol with a name like MinimongoTest.)